src/silx/math/_colormap.pyx

```diff
@@ -247,7 +248,7 @@ cdef class SqrtNormalization(Normalization):
         return sqrt(value)
 
     cdef double revert_double(self, double value, double vmin, double vmax) nogil:
-        return value**2
+        return value*value
```

Comment:
Not needed but I'd expect `*` to be faster than `pow`

Reply:
makes sense